### PR TITLE
Update readme

### LINKED ISSUE #82: Github Actions table in README
markdown table in the readme explain what Action we have with the following columns

- badge (from #71)
- name
- triggers
- results

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
Fixes #82 #71 
Also resloves PR #105 

#### Proposed Changes

- Added repository related tags like license, issues, forks
- Added CI action badges with Github Actions table

### Added repository related tags like license, issues, forks
Ss1
<br/>

### Added CI action badges with Github Actions table
Ss2

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
@eddiejaoude Please review this PR

File: README.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
-# Discord bot using Stargate
 
 
+<h1 align="center">
+  :robot: Discord bot using Stargate :robot:
+</h1>
+
+
```
Comment (eddiejaoude) [inline README.md:7]: Probably only a single blank line needed here

File: README.md [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+<div align="center">
+
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/eddiejaoude/stargate.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/eddiejaoude/stargate/alerts/)
+![Issues](https://img.shields.io/github/issues/eddiejaoude/stargate)
+![Forks](https://img.shields.io/github/forks/eddiejaoude/stargate)
+![Stars](https://img.shields.io/github/stars/eddiejaoude/stargate)
+![Docs status](https://img.shields.io/github/license/eddiejaoude/stargate)
 
+</div>
```
Comment (eddiejaoude) [inline README.md:16]: Probably more readable to have a blank line here

**Comment by eddiejaoude** [comment]
Looks good 👍 , I left a few small comments

**Comment by kaiwalyakoparkar** [comment]
@eddiejaoude Changes made have a look at it!

File: README.md [hunk 3]
```diff
@@ -23,6 +34,17 @@
...
+| [![Docs status](https://github.com/eddiejaoude/stargate/workflows/Docs/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/docs.yml) | Docs status | Pushes to doc/ repository | checkout source files, duild doc files, deploy |
+| [![Labels status](https://github.com/eddiejaoude/stargate/workflows/Import%20labels/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/labels.yml) | Labels status | Pushes to doc/labels.json | checkout source files, install deps, import labels |
+| [![Node CI status](https://github.com/eddiejaoude/stargate/workflows/Node.js%20CI/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/node-ci.yml) | Node CI status | Pushes & Pull Request | Checkout source files, install dependencies, lint source files, verify build, run tests |
+| [![Stale issues and PRs status](https://github.com/eddiejaoude/stargate/workflows/Mark%20stale%20issues%20and%20pull%20requests/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/stale.yml) | Stale issues and PRs status | Scheduled at `30 1 * * *` | Updates issue / PR messages & labels |
```
Comment (MitanshuShaBa) [inline README.md:46]: Probably instead on cron syntax writing every 30 min will be more clear

Comment (kaiwalyakoparkar) [reply README.md]: Done :+1:

**Review by MitanshuShaBa — APPROVED** [review]
Looks good 👍

File: README.md [hunk 4]
```diff
@@ -23,6 +34,17 @@
...
+| [![Docs status](https://github.com/eddiejaoude/stargate/workflows/Docs/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/docs.yml) | Docs status | Pushes to doc/ repository | checkout source files, duild doc files, deploy |
+| [![Labels status](https://github.com/eddiejaoude/stargate/workflows/Import%20labels/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/labels.yml) | Labels status | Pushes to doc/labels.json | checkout source files, install deps, import labels |
+| [![Node CI status](https://github.com/eddiejaoude/stargate/workflows/Node.js%20CI/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/node-ci.yml) | Node CI status | Pushes & Pull Request | Checkout source files, install dependencies, lint source files, verify build, run tests |
+| [![Stale issues and PRs status](https://github.com/eddiejaoude/stargate/workflows/Mark%20stale%20issues%20and%20pull%20requests/badge.svg)](https://github.com/eddiejaoude/stargate/actions/workflows/stale.yml) | Stale issues and PRs status | Iterval of every 30 minutes | Updates issue / PR messages & labels |
```
Comment (eddiejaoude) [inline README.md:46]: Spelling mistake of `Iterval`, but this still adds lots of value, so I will merge and a new PR can be added to fix this 🤓

**Review by eddiejaoude — APPROVED** [review]
Great work and collaboration 👍

**Pull request merged**